site/_posts/2017-04-10-Announcing-Euwe-2.1.html.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Announcing Euwe-2.1
+author: jprause
+date: 2017-04-10 08:35:39 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+We've just built Euwe-2.1. This release resolves the issue in Euwe-2, where dropdowns and trees don't work in the UI. There is also a manual fix found [here](http://talk.manageiq.org/t/euwe-2-broken-ui-fix/2221).
+
+| Appliance | Format | Size |
+| --------- | ------ | ---- |
+| [Amazon EC2](http://releases.manageiq.org/manageiq-ec2-euwe-2.1.vhd) | ec2 | 3.3 GB |
```

Comment:
I know we talked about this, but I just realized I added as "Amazon AWS EC2" to the download page.. Whichever it is, I think it should be consistent.